### PR TITLE
[Etc] Update gradle build info to Marshmallow (23)

### LINKED ISSUE #97: Upgrade target SDK to Marshmallow (23)
Don't we have to keep up, eh?

### PR DESCRIPTION
**Pull request by wiseodd**
Closes #97

### CODE REVIEW

**Comment by timotiusnc** [comment]
Steps:
1. Open a news.
2. Collapse a comment.

Expectation:
3. Collapsed.

Reality:
3. Gone.

File: app/src/main/java/io/pumpkinz/pumpkinreader/NewsDetailFragment.java [hunk 1]
```diff
@@ -77,6 +77,10 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
         newsDetail = (RecyclerView) view.findViewById(R.id.news_detail);
+        newsDetail.getItemAnimator().setRemoveDuration(100);
+        newsDetail.getItemAnimator().setChangeDuration(0);
```
Comment (timotiusnc) [inline app/src/main/java/io/pumpkinz/pumpkinreader/NewsDetailFragment.java:81]: Why is `change duration` the only one with 0 seconds duration, not 100 like the others?

Comment (wiseodd) [inline app/src/main/java/io/pumpkinz/pumpkinreader/NewsDetailFragment.java:81]: `setChangeDuration` is when we touch the comment item.
I set it to 0 to make it snappier.
The default value will make the comment item blinks.

Comment (timotiusnc) [inline app/src/main/java/io/pumpkinz/pumpkinreader/NewsDetailFragment.java:81]: Oh, all right.

**Pull request merged**